rancid-tomatillos/src/Components/Header/Header.js

```diff
@@ -1,27 +1,37 @@
 import React from 'react';
 import './Header.scss';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
+import { Route, Link } from 'react-router-dom';
 import { removeUser, updateLoggedIn } from '../../actions/index';
 
 
-const Header = ({ data, isLoggedIn, removeUser, updateLoggedIn }) => {
+const Header = ({ data, isLoggedIn, removeUser, updateLoggedIn, path }) => {
   let log;
+  let gradient;
+  const index = Math.floor(Math.random() * Math.floor(20));
+  const imageUrl = data.movies[index].backdrop_path;
   if (isLoggedIn) {
-    log = <button onClick={() => {updateLoggedIn(isLoggedIn); removeUser();}} className='header-btn-logout'>Log Out</button>
+    log = <Link to='/' className='login-link'>
+    <button onClick={() => {updateLoggedIn(isLoggedIn); removeUser();}} className='header-btn-logout'>Log Out</button>
+    </Link>
   } else {
     log = <Link to='/login' className='login-link'>
     <button className='header-btn'>Log In</button>
     </Link>
   }
-  const index = Math.floor(Math.random() * Math.floor(20));
+  if (path.includes('movies')) {
+    gradient = `0.9`;
+  } else {
+    gradient = '0';
+  }
   return (
```

Comment:
I do appreciate the gradient, and think it is a step in the right direction. But kinda feels like we are duct taping a solution here. Overall, I find the effectiveness of the random header image to be hit or miss. Sometimes it lines up perfect and looks great, other times its a close up of a guys nose. I know you put a lot of work into this, so feel bad asking you to revamp, but maybe we could add a collection of images of classic films that we know will line up correctly? And that way we don't run into those situations where the header matches the background and the main poster. Just feel like we can find a better solution....but your effort on this is appreciated!

Reply:
I totally agree with this as well. The way it is currently definitely seems to be a bit buggy with the images provided and doesn't always look the best depending on the image that is currently selected. I think given more time there could definitely be a better way to format this. Though, I do think within the scope of this project that the way it is know is definitely fine for the time being. I think our time would be better spent first finishing out all testing and getting iteration 5 working. We could add an issue on GH for a potential refactoring of how that header image is being determined. If we are able to finish the above and still have time then that would be a good thing to explore as an extension type thing. Also, if you have some ideas on how to improve it and want to explore that over break then by all means feel free to go for it.